vcsh

```diff
@@ -170,6 +175,8 @@ clone() {
 	git checkout -b "$VCSH_BRANCH" || return $?
 	git config branch."$VCSH_BRANCH".remote origin
 	git config branch."$VCSH_BRANCH".merge  refs/heads/"$VCSH_BRANCH"
+	# BSD wc emits spaces.
```

Comment:
Yes. So? I don't think quoting this hurts, `[ " 2 " -eq 2 ]` tests the number just fine, we can probably go with it's recomendation and not worry about the override.

Reply:
https://github.com/RichiH/vcsh/pull/293/commits/b66ed4e723ff58f4ad39477fd3257caff5ba7fe0 behaves correctly.

Reply:
https://github.com/RichiH/vcsh/pull/293/commits/b66ed4e723ff58f4ad39477fd3257caff5ba7fe0 behaves correctly and is not nagged at by shellcheck.